lua/autorun/server/bounties.lua

```diff
@@ -1,8 +1,12 @@
 
+BountyLimit = {}
```

Comment:
Generally do this like:
BountyLimit = BountyLimit or {}

That way if the file is reloaded it doesn't reset the BountyLimit